data/data.go

```diff
@@ -31,12 +31,113 @@ type EffxYaml struct {
 	FilePath string
 }
 
+// converts effxhq/backeffx to backeffx
+// needed for github env var
+func parseOutOwnerName(repoName string) string {
+	if strings.Contains(repoName, "/") {
+		result := strings.Split(repoName, "/")
+		if len(result) > 1 {
+			return result[1]
+		}
+	}
+
+	return repoName
+}
+
+// given a absolute path (example: /runner/folder/root/data/effx.yaml)
+// and given a workingDir (example: root)
+// it will return data/effx.yaml
+func parseRelativePathFromAbsolutePath(absoluteDir, workingDir string) string {
+	// if working directory does not end with a slash, add it
+	if strings.LastIndex(workingDir, "/") != len(workingDir)-1 {
+		workingDir += "/"
+	}
+
+	res := strings.Split(absoluteDir, workingDir)
+	if len(res) > 1 {
+		return res[1]
+	}
+	return absoluteDir
+}
+
+// given an absolute path, this function will find the root directory
+// (directory with a .git file)
+func findGitRootDirectory(absoluteDir string) (string, error) {
+	pathDir := filepath.Dir(absoluteDir)
+
+	for pathDir != "" {
+		pathDir = filepath.Clean(pathDir)
+		files, err := ioutil.ReadDir(pathDir)
+
+		if err != nil {
+			return "", err
+		}
+
+		for _, file := range files {
+			if !file.IsDir() {
+				if strings.Contains(file.Name(), ".git") {
+					return filepath.Base(pathDir), nil
+				}
+			}
+		}
+
+		pathDir = filepath.Join(pathDir, "..")
+	}
+
+	return absoluteDir, nil
+}
+
+func getRepoName() string {
```

Comment:
Do you think it would be easier to parse the repo name from the git URL? You already are parsing it in other parts of the PR so that may be a CI agnostic way to approach this?